tensorflow/compiler/mlir/tosa/transforms/legalize_tfl.cc

```diff
@@ -2267,6 +2298,52 @@ LogicalResult ConvertTFLPadOp::matchAndRewrite(
   return success();
 }
 
+LogicalResult ConvertTFLResizeBilinearOp::matchAndRewrite(
+    Operation* op, PatternRewriter& rewriter) const {
+  auto tfl_resize_op = cast<TFL::ResizeBilinearOp>(op);
+
+  RankedTensorType output_type =
+      tfl_resize_op.getResult().getType().dyn_cast<RankedTensorType>();
+  // Not a ranked tensor output
+  if (!output_type) return failure();
+
+  bool align_corners = tfl_resize_op.align_cornersAttr().getValue();
```

Comment:
Nit: I don't think these separate bools are carrying their weight. I would just inline them into the call to `convertResizeOp`.

Reply:
Thanks will do. I'd assume same comment for "half_pixel_centers" 1 line below?